packages/react-components/react-input/src/components/Input/useInputStyles.ts

```diff
@@ -15,32 +15,9 @@ export const inputClassNames: SlotClassNames<InputSlots> = {
 };
 
 // TODO(sharing) use theme values once available
```

Comment:
Can we remove this comment now?

Reply:
There is the leftover contentSizes[400] so I left that comment there.